lib/utils/documentdb.ts

```diff
@@ -469,6 +469,10 @@ export function replaceDocument<T>(
       documentUri.uri,
       document,
       {
+        accessCondition: {
+          condition: document._etag,
```

Comment:
I don't understand what's happening here, `document` is the *new* document, how can its `etag` match the `etag` of the existing document?